src/EventStore.Plugins/Authentication/IAuthenticationProvider.cs

```diff
@@ -13,5 +15,26 @@ public interface IAuthenticationProvider {
 		/// </summary>
 		/// <param name="authenticationRequest"></param>
 		void Authenticate(AuthenticationRequest authenticationRequest);
+
+		/// <summary>
+		///     Return a unique name used to externally identify the authentication provider.
+		/// </summary>
+		string GetName();
```

Comment:
Is there a reason not to make this a readonly property?

Reply:
Thanks, I've made the changes